modules/contributor-guide/partials/ref_che-theia-plug-in-api.adoc

```diff
@@ -40,7 +40,7 @@ The available resources of the `theia` {namespace} are described in the link:htt
 
 NOTE: Many APIs in the {namespace} are experimental and can change in the future.
 
-Available resources of the `che` {orch-namespace} are described in the link:https://raw.githubusercontent.com/eclipse/che-theia/{prod-ver-patch}/extensions/eclipse-che-theia-plugin/src/che.d.ts[`che.d.ts`] file in the link:https://github.com/eclipse-che/che-theia[Che-Theia repository].
+Available resources of the `che` {orch-namespace} are described in the `extensions/eclipse-che-theia-plugin/src/che.d.ts` file in the link:https://github.com/eclipse-che/che-theia[Che-Theia repository].
```

Comment:
Removing reference to https://github.com/eclipse-che/che-theia/blob/main/extensions/eclipse-che-theia-plugin/src/che.d.ts as it is now empty.

```suggestion
```